airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java

```diff
@@ -185,7 +185,10 @@ public void trackSync(Job job, JobState jobState) {
     final StandardDestinationDefinition destinationDefinition = configRepository.getStandardDestinationDefinition(destinationDefinitionId);
     metadata.put("connector_destination", destinationDefinition.getName());
     metadata.put("connector_destination_definition_id", destinationDefinition.getDestinationDefinitionId());
-
+    final String[] imageTag = destinationDefinition.getDockerImageTag().split(":");
```

Comment:
Should it be the docker image (version) tag only? or the full docker image name instead?

Reply:
i think we should do one of the following. 
1. full docker image name with tag e.g. `connector_destination_version ` = `airbyte/postgres-source:0.2.0` 
2. pull the docker image name and tag but put them in separate fields `connector_destination_image_name` = `airbyte/postgres-source`and `connector_destination_image_version`= `0.2.0`

The one thing I feel strongly about is just saving the tag is enough enough.

Reply:
Seems like John is ok with just the version tag `0.2.0`